docs/pages/configuration/schema.mdx

```diff
@@ -16,6 +16,7 @@ max_wal_history = 10000
 byron_path = "byron.json"
 shelley_path = "shelley.json"
 alonzo_path = "alonzo.json"
+conway_path = "conway.json"
```

Comment:
didn't realize I need to update this section as well: https://dolos.txpipe.io/configuration/schema#genesis-section